CHANGELOG.md

```diff
@@ -41,9 +57,17 @@ Development for the Beta Release is currently in progress, features implemented
 - Delete channel
 - View people in a channel
 - Add/remove people from a channel
+- Loading screen when opening channels 
```

Comment:
LB maybe for the `Unread posts above/below indicator` we should mention that only works on ios for now